Gemfile

```diff
@@ -30,9 +30,5 @@ if RUBY_VERSION >= '1.8.7' and RUBY_VERSION < '1.9'
   gem 'rake', '~> 10.0'
 end
 
-if RUBY_VERSION < '2.0'
-  # json 2.x requires ruby 2.0. Lock to 1.8
-  gem 'json', '~> 1.8'
-  # json_pure 2.0.2 requires ruby 2.0. Lock to 2.0.1
-  gem 'json_pure', '= 2.0.1'
-end
+gem 'json',      '<= 1.8'   if RUBY_VERSION < '2.0.0'
+gem 'json_pure', '<= 2.0.1' if RUBY_VERSION < '2.0.0'
```

Comment:
@ghoneycutt I don't yet understand all the secrets of the Gemfile. But with this I got that beast to work with me again. Please have a second look at this. I quite like it because it's easy to read in my opinion.